_data/projects/jmeta

```diff
@@ -0,0 +1,12 @@
+name: JMeta
+desc: JMeta is a free software project, written in Java, aiming to provide a search platform and a framework for building fully-decentralized P2P search applications.
+site: https://gitlab.ouahpiti.info/meta/jmeta
```

Comment:
There's an issue with the HTTPS configuration on this server, so you might have problems with visitors accessing the site...